script.js

```diff
@@ -1,329 +1,347 @@
-
-    // Append popupLiveButtonBox to popupButtonsBox
-    popupButtonsBoxs[i].appendChild(popupLiveButtonBoxs[i]);
-
-    // Create PopupSourceButton and append it to popupButtonBox
-    popupSourceButtonBoxs[i] = document.createElement('div');
-    popupSourceButtonBoxs[i].classList.add('popupSourceButtonBox', 'flex-row');
-    popupSourceLinks[i] = document.createElement('a');
-    popupSourceLinks[i].href = projects[i].codeSource;
-    popupSourceLinks[i].target = '_blank';
-    popupSourceLinks[i].rel = 'noopener noreferrer';
-    popupSourceLinks[i].innerHTML = 'See source';
-    popupSourceButtonBoxs[i].appendChild(popupSourceLinks[i]);
-    let sourceButtonIcon = document.createElement('img');
-    sourceButtonIcon.src = './Icons/source.png';
-    sourceButtonIcon.alt = 'source icon';
-    popupSourceButtonBoxs[i].appendChild(sourceButtonIcon);
-    
-    // Append popupSourceButtonBox to popupButtonsBox
-    popupButtonsBoxs[i].appendChild(popupSourceButtonBoxs[i]);
-
-
-// Mobile screen size popup window content creation End
-
-// Open and close popup window 
-    // Create close mobile popup window
-    projectButtons[i].addEventListener('click', function(){
-        popupWindowContent.classList.add('flex-column');
-        if(window.innerWidth < 768){
-            popupWindowContent.append(closeMobilePopupWindow, popupImageBoxs[i], popupTitleBoxs[i], popupTechList[i], projectDescriptions[i], popupButtonsBoxs[i]);
-        } else{
-            let popupTitleAndButtonsBox = document.createElement('div');
-            popupTitleAndButtonsBox.classList.add('popupTitleAndButtonsBox', 'flex-row');
-            popupTitleAndButtonsBox.append(popupTitleBoxs[i], popupButtonsBoxs[i]);
-            popupWindowContent.append(closeMobilePopupWindow, popupImageBoxs[i], popupTitleAndButtonsBox, popupTechList[i], projectDescriptions[i]);
-        }
-        document.body.style.position = 'fixed';
-        popupWindow.classList.add('show');    
-    });
-    closeMobilePopupWindow.addEventListener('click', function(){
-        popupWindowContent.innerHTML = '';
-        document.body.style.position = '';
-        popupWindow.classList.remove('show');
-    });
-}
\ No newline at end of file
+// Contact Form Validation Start
+const contactForm = document.querySelector('.contactForm');
+const contactFormEmail = document.querySelector('#email');
+const contactFormList = document.querySelector('.contactFormList');
+const errorMsgContainer = document.createElement('li');
+errorMsgContainer.className = 'errorMsgContainer';
+contactFormList.appendChild(errorMsgContainer);
+
+contactForm.addEventListener('submit', (event) => {
+  const emailAddress = contactFormEmail.value.replace(/[^a-zA-Z ]/g, '');
+  // stop form submission
+  event.preventDefault();
+
+  // Validate the form
+  if (emailAddress !== emailAddress.toLowerCase()) {
+    errorMsgContainer.innerHTML = 'Invalid Email Format. Please enter a correct email address (lowercase only)!';
```

Comment:
[OPTIONAL] Well done on showing the error ❌ but the user only needs a few seconds to see it, consider removing it after a few seconds, you could do that by adding this 👇🏽 

```
    setTimeout(() => {
      errorMsgContainer.innerHTML = '';
    }, 3000);
```